src/ripple/app/ledger/Ledger.cpp

```diff
@@ -452,7 +452,7 @@ bool Ledger::saveValidatedLedger (bool current)
         "INSERT OR REPLACE INTO Ledgers "
         "(LedgerHash,LedgerSeq,PrevHash,TotalCoins,ClosingTime,PrevClosingTime,"
         "CloseTimeRes,CloseFlags,AccountSetHash,TransSetHash) VALUES "
-        "('%s','%u','%s','%s','%u','%u','%d','%u','%s','%s');");
+        "('%s','%u','%s','%u','%u','%u','%d','%u','%s','%s');");
```

Comment:
I thought it was, although now it occurs to me that it may be wrong: `%u` is probably not the right type for `std::uint64_t`. I'll roll this bit back, nice catch.